resources/models/reservation.py

```diff
@@ -75,10 +76,27 @@ class Meta:
     def __str__(self):
         return "%s -> %s: %s" % (self.begin, self.end, self.resource)
 
+    def clean(self):
+        if self.end <= self.begin:
```

Comment:
Should you call `super()` before doing the rest of validation (to make sure that `self.resource` is set etc.)?